### PR TITLE
Temporarily disable coverage in Travis builds

### PR DESCRIPTION
**Pull request by tanabarr**
Running coverage plug-in for pytest seems to be broken.

Temporarily remove running coverage to create the reports while we find a fix.

https://travis-ci.org/intel-hpdd/iml-common/builds/253755003
https://coverage.readthedocs.io/en/coverage-4.4.1/changes.html
https://bitbucket.org/ned/coveragepy/commits/74e2f7c366a2

### CODE REVIEW

File: tests/lib/test_util.py [hunk 1]
```diff
@@ -36,6 +32,17 @@ def test_file(self):
             self.assertEqual(mock_os_chown.call_count, 1)
             mock_os_chown.assert_called_with(filename, magic_stat_mock.st_uid, magic_stat_mock.st_gid)
 
+
+class DisableThreadsTestCase(ImlUnitTestCase):
+
+    def setUp(self):
```
Comment (jgrund) [inline tests/lib/test_util.py]: Is this related to the coverage not being collected?

Comment (tanabarr) [inline tests/lib/test_util.py]: it is a tidy up of the test code, so not directly related
